app/services/repository_service.rb

```diff
@@ -1,17 +1,21 @@
-# frozen_string_literal: true
-
 class RepositoryService
-  attr_reader :user
+  def self.call(user, github_gateway, query = nil)
```

Comment:
to jest bardzo nie fajne, bo po pierwsze send, a po drugie jest niepotrzebnym sugar syntaxem.

Repository to jest w ogole design pattern bro - https://medium.com/@pererikbergman/repository-design-pattern-e28c0f3e4a30.

Imo masz 2 opcje
1) zmienilbym naming na fetchuserresitories, najlepiej w namespace Github
2) zrob z tego pelnoprawne repo 1 metoda publiczna `fetch_repositories`